feature/sign/src/main/java/com/moneymong/moneymong/feature/sign/view/SignUpUniversity.kt

```diff
@@ -0,0 +1,223 @@
+import com.moneymong.moneymong.design_system.theme.White
+import com.moneymong.moneymong.feature.sign.SignUpContent
+import com.moneymong.moneymong.feature.sign.sideeffect.SignUpSideEffect
+import com.moneymong.moneymong.feature.sign.sideeffect.SignUpUniversitySideEffect
+import com.moneymong.moneymong.feature.sign.state.SignUpUniversityState
+import com.moneymong.moneymong.feature.sign.util.AgencyType
+import com.moneymong.moneymong.feature.sign.viewmodel.SignUpUniversityViewModel
+import com.moneymong.moneymong.feature.sign.viewmodel.SignUpViewModel
+import org.orbitmvi.orbit.compose.collectAsState
+import org.orbitmvi.orbit.compose.collectSideEffect
+
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
+@Composable
+fun SignUpUniversity(
+    navigateToLedger : () -> Unit,
+    navigateToAgency : () -> Unit,
+    navigateUp : () -> Unit,
+    agencyName : String,
+    agencyType: AgencyType,
+    viewModel: SignUpUniversityViewModel = hiltViewModel()
+){
+    val state = viewModel.collectAsState().value
+
+    BackHandler {
+        navigateUp()
+    }
+
+    if(state.visibleError){
+        ErrorScreen(
+            modifier = Modifier.fillMaxSize(),
+            message = state.errorMessage,
+            onRetry = {
+                viewModel.visibleErrorChanged(false)
+            }
+        )
+    }
+    else if(state.visiblePopUpError){
+        ErrorDialog(
+            message = state.popUpErrorMessage,
+            onConfirm = {
+                viewModel.visiblePopUpErrorChanged(false)
+            }
+        )
+    }
+    else {
+        Scaffold(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(White)
+                .padding(horizontal = MMHorizontalSpacing),
+            topBar = {
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(44.dp)
+                        .background(White),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.Start
+                ) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_chevron_left),
+                        contentDescription = null,
+                        modifier = Modifier
+                            .size(24.dp)
+                            .background(White)
+                            .noRippleClickable {
+                                navigateUp()
+                            },
+                        tint = Gray07
+                    )
+                }
+            },
+            content = {
+                SignUpUniversityContent(
+                    navigateToLedger = navigateToLedger,
+                    agencyName = agencyName,
+                    agencyType = agencyType,
+                    viewModel = viewModel,
+                    state = state
+                )
+            }
```

Comment:
"어디 학교 교내 동아리인가요?" Text 가 TopBar 에 가려져서 보이지 않아요!
<image width =400, src = "https://github.com/user-attachments/assets/9dd163d9-357e-4347-a41d-a41f4541aea7"/>

```suggestion
            content = { paddingValues ->
                SignUpUniversityContent(
                    modifier = Modifier.padding(paddingValues),
                    navigateToLedger = navigateToLedger,
                    agencyName = agencyName,
                    agencyType = agencyType,
                    viewModel = viewModel,
                    state = state
                )
            }
```